### PR TITLE
updates SoftEdgeWidget to include actionId

### PR DESCRIPTION
**Pull request by chloealee**
### What does this PR do?

This PR adds the `actionId` to the SoftEdgeWidget for the user to add this when creating a new SoftEdge action to the page and to send this information in the URL to SoftEdge.

### What are the relevant tickets/cards?

Refs first task in [Pivotal ID #165974086](https://www.pivotaltracker.com/n/projects/2019313/stories/165974086)

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: contentful/content-types/softEdgeWidgetAction.js [hunk 1]
```diff
@@ -1,64 +1,82 @@
 module.exports = function(migration) {
   const softEdgeWidgetAction = migration
-    .createContentType('softEdgeWidgetAction')
-    .name('SoftEdgeWidgetAction')
+    .createContentType("softEdgeWidgetAction")
```
Comment (mendelB) [inline contentful/content-types/softEdgeWidgetAction.js]: Do ya know why these single quotes got replaced with doubles? 🤔I feel like our other scripts use singles tho obviously not a massive deal lol

File: contentful/content-types/softEdgeWidgetAction.js [hunk 2]
```diff
@@ -1,64 +1,82 @@
...
-  softEdgeWidgetAction.changeEditorInterface('internalTitle', 'singleLine', {
+  softEdgeWidgetAction
+    .createField("actionId")
+    .name("Action Id")
+    .type("Integer")
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        unique: true
```
Comment (mendelB) [inline contentful/content-types/softEdgeWidgetAction.js]: I know we've traditionally added the `unique` validation to our Action ID fields, but I *feel* like I recall @DFurnes or @weerd making the case for these not to be unique 🤔 I think the argument was something along the lines of potentially having the ability to make two separate UI representations of the same Action on say a Campaign and a separate article page or the like. But might be worth being consistent with this until we remove the validation across actions if necessary?

File: contentful/content-types/softEdgeWidgetAction.js [hunk 3]
```diff
@@ -1,64 +1,82 @@
...
 
-  softEdgeWidgetAction.changeEditorInterface('softEdgeId', 'numberEditor', {
+  softEdgeWidgetAction.changeEditorInterface("softEdgeId", "numberEditor", {
     helpText:
-      'The ID of the SoftEdge action, given upon creation in SoftEdge. When you click "Publish Now", a URL will appear in the "See the Action" box. The SoftEdge ID is the last digit in the URL. e.g. SoftEdge ID = 3 in http://www.congressweb.com/dosomething/3',
+      'The ID of the SoftEdge action, given upon creation in SoftEdge. When you click "Publish Now", a URL will appear in the "See the Action" box. The SoftEdge ID is the last digit in the URL. e.g. SoftEdge ID = 3 in http://www.congressweb.com/dosomething/3'
+  });
+
+  softEdgeWidgetAction.changeEditorInterface("actionId", "numberEditor", {
+    helpText: "The Action ID associated with this action in Rogue."
```
Comment (mendelB) [inline contentful/content-types/softEdgeWidgetAction.js]: Would it be worth adding the link to actions in the help text as we did here? https://github.com/DoSomething/phoenix-next/blob/273a35eb8011e173333ab6744ee3db2151a93e63/contentful/content-types/postGallery.js#L81-L84

File: contentful/content-types/softEdgeWidgetAction.js (hunk 1 above)
Comment (chloealee) [reply contentful/content-types/softEdgeWidgetAction.js]: I don't know why that happened! I can update those back to change to single quotes though!

Comment (weerd) [reply contentful/content-types/softEdgeWidgetAction.js]: When generating the migration, it generates them with double quotes. But then when saving the file Prettier goes in the turns them all into single quotes. @chloealee is your Prettier extension working correctly? Does it modify your code when you save the file?

File: contentful/content-types/softEdgeWidgetAction.js (hunk 3 above)
Comment (weerd) [reply contentful/content-types/softEdgeWidgetAction.js]: Seems useful! Let's add that 💃

File: contentful/content-types/softEdgeWidgetAction.js (hunk 2 above)
Comment (weerd) [reply contentful/content-types/softEdgeWidgetAction.js]: Yeah... hmmm, maybe lets be consistent for now so if we decide to update we will know to do it to all the actions.

Comment (weerd) [reply contentful/content-types/softEdgeWidgetAction.js]: But defs good note and something to swing back to...

File: contentful/content-types/softEdgeWidgetAction.js (hunk 1 above)
Comment (chloealee) [reply contentful/content-types/softEdgeWidgetAction.js]: @weerd hm it must not be working! I just manually changed all these back though - lmk if there is anything that I missed!

File: contentful/content-types/softEdgeWidgetAction.js (hunk 3 above)
Comment (chloealee) [reply contentful/content-types/softEdgeWidgetAction.js]: love that idea, thanks @mendelB ! Added!

File: contentful/content-types/softEdgeWidgetAction.js (hunk 1 above)
Comment (weerd) [reply contentful/content-types/softEdgeWidgetAction.js]: Gah! We definitely need to get it working on your editor. It'll save you from lots of little headaches! I can help you next week with it. Sometimes it's an issue with the Prettier package in Sublime knowing the path to NodeJS. I've had to fix it in the past. If you bring up the Sublime console (Control + `), you might see an error every time you save a file...

Comment (DFurnes) [reply contentful/content-types/softEdgeWidgetAction.js]: Hmm, it seems like the git hooks may not be working anymore... let me take a look. (But also absolutely grab the [Prettier extension](https://prettier.io/) for your editor if you haven't yet!)

Comment (chloealee) [reply contentful/content-types/softEdgeWidgetAction.js]: will do thank you both!

**Pull request merged**